src/functions-and-arrays.js

```diff
@@ -1,41 +1,141 @@
 // Iteration #1: Find the maximum
-function maxOfTwoNumbers() {}
+function maxOfTwoNumbers(n1,n2) {
+  if (n1 > n2){
+    return n1;
+  } else {
+    return n2;
+  }
+}
 
 
 
 // Iteration #2: Find longest word
 const words = ['mystery', 'brother', 'aviator', 'crocodile', 'pearl', 'orchard', 'crackpot'];
 
-function findLongestWord() {}
+function findLongestWord(words) {
+  let longestWord;
+  if (words.length === 0){
+    return null;
+
+  } else {
+    for (let i=0; i < words.length; i++){
+      let word = words[i];
+      if (!longestWord || longestWord.length < word.length){
+      longestWord = word;
+  }
+    } 
+    return longestWord;
+  }
+}
 
 
 
 // Iteration #3: Calculate the sum
 const numbers = [6, 12, 1, 18, 13, 16, 2, 1, 8, 10];
 
-function sumNumbers() {}
+function sumNumbers(numbers) {
+  let sum = 0
+  if (numbers.length === 0){
+    return 0;
+  } else {
+    for (let i = 0; i<numbers.length; i++){
+      let number = numbers[i]
+      sum += number;
+      }
+  }
+  return sum
```

Comment:
```suggestion
  if (numbers.length === 0){
    return 0;
  } 
  return sum(numbers);
```

Ya que te has hecho el metodo sum! pues vamos a usarlo! :D